Misc/NEWS.d/next/Windows/2020-09-24-23-09-40.bpo-39107.GbUZvD.rst

```diff
@@ -0,0 +1 @@
+Updated Tcl and Tk to 8.6.10 in Windows installer
```

Comment:
```suggestion
Updated Tcl and Tk to 8.6.10 in Windows installer.
```

Reply:
I have made the requested changes; please review again.